examples/telnet/as400.js

```diff
@@ -0,0 +1,127 @@
+
+/**
+ * Domotz Custom Driver 
+ * Name: AS400
```

Comment:
This should likely be specified as AS400 (IT)

And we need to consider adding an EN version as the main one